src/Select.js

```diff
@@ -562,6 +562,32 @@ export default class Select extends Component<Props, State> {
     });
   }
 
+  changeFocusedOption(opt: string){
+    if(opt == null){
+       return;
+    }
+    const { menuOptions } = this.state;
+    const options = menuOptions.focusable;
+    
+    if(!options.length){
+       return;
+    }
+    let index = -1;
+    let i = 0;
+    for(i = 0; i < options.length; i++){
```

Comment:
Suggestion: findIndex already does this - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex - providing a predicate as argument

Reply:
On the other side, why aren't we using the `find` function? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find .

Reply:
Right now I only send the label of the option to the method so `find` would not work, but `findIndex` is indeed the right way

Reply:
If you don't need the index but just the element, I would suggest using the `find`